app/src/components/Tabs/Activity/activity-tab.tsx

```diff
@@ -370,13 +382,17 @@ const ActivityTab: FC<ActivityTabProps> = ({
                           <DirectMeasureTable
                             t={t}
                             activityData={activityValues}
+                            onDeleteActivity={onDeleteActivity}
+                            onEditActivity={onEditActivity}
                           />
                         ) : (
                           <ActivityAccordion
                             t={t}
                             activityData={activityValues}
                             showActivityModal={onAddActivityModalOpen}
                             methodologyId={methodology?.id}
+                            // onDeleteActivity={onDeleteActivity}
+                            // onEditActivity={onEditActivity}
```

Comment:
Can you also implement these for the ActivityAccordion [sic] or add a TODO comment here and a Jira ticket for it?